### PR TITLE
Add migrations tb mateials and student materials

### LINKED ISSUE #6: Create Migration
Screenshot 2024-07-12 at 14 56 15

### PR DESCRIPTION
**Pull request by indrafani03**
Im already add migration for table materials and student_materials

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_PORT=5432
```
Comment (jacky-htg) [inline .env]: file .env harus di gitignore, agar credential tidak terekspose.

- hapus file .env
- masukkan ke gitignore
- buat lagi file env

**Review by jacky-htg — CHANGES_REQUESTED** [review]
.env file should be add on gitignore

File: .env (hunk 1 above)
Comment (indrafani03) [reply .env]: done mas

**Review by jacky-htg — APPROVED** [review]
LGTM

**Pull request merged**